teamwork/templates/projects/view_project.html

```diff
@@ -137,63 +137,85 @@
+                            &nbsp;
+                            <a href="{% url 'profile' project.ta.username %}">
+                                <span class="label label-info">
+                                    {{ project.ta.username }}
+                                </span>
+                            </a>
+                            &nbsp;&nbsp;&nbsp;&nbsp;
+                            <strong>Members</strong>
+                            &nbsp;
+                            {% for mem in project.members.all %}
+                            {% if mem.username != null %}
+                            <a href="{% url 'profile' mem.username %}">
+                            <span class="label label-primary">
+                                {{ mem.username }}
                             </span>
-                        </a>
-                    </p>
-                    <p><strong>Members</strong>
-                        {% for mem in project.members.all %}
-                        {% if mem.username != null %}
-                        <a href="{% url 'profile' mem.username %}">
-                        <span class="label label-primary">
-                            {{ mem.username }}
-                        </span>
-                        </a>
-                        {% endif %}
-                        {% endfor %}
-                    </p>
-                    <p><strong>Desired Skills</strong>
-                        {% if project.desired_skills != none %}
-                        {% for skill in project.desired_skills.all %}
-                        <span class="label label-warning">
-                            {{ skill.skill }}
-                        </span>
-                        {% endfor %}
-                        {% endif %}
-                    </p>
-
-                    <p><strong>Meeting Location</strong>
-                            {{ project.ta_location }}
-                    </p>
-
-                    <p><strong>Meeting Time</strong>
-                            {{ project.ta_time }}
-                    </p>
+                            </a>
+                            &nbsp;
+                            {% endif %}
+                            {% endfor %}
+                            &nbsp;&nbsp;&nbsp;&nbsp;
+                            <strong>Scrum Master</strong>
+                            &nbsp;
+                            <a href="{% url 'profile' project.scrum_master.username %}">
+                                <span class="label label-info">
+                                    {{ project.scrum_master.username }}
+                                </span>
+                            </a>
+                            &nbsp;&nbsp;&nbsp;&nbsp;
+                            <strong>Meeting Location</strong>
+                            &nbsp;
+                            {% if project.ta_location == "" %}
+                                N/A
```

Comment:
do we have to put these N/A in an html tag

Reply:
They're just filler text if the field is empty, they are fine being out